crates/revm/revm-inspectors/src/tracing/js/mod.rs

```diff
@@ -277,9 +277,28 @@ where
     fn initialize_interp(
         &mut self,
         _interp: &mut Interpreter,
-        _data: &mut EVMData<'_, DB>,
+        data: &mut EVMData<'_, DB>,
         _is_static: bool,
     ) -> InstructionResult {
+        let precompiles: hashbrown::HashMap<[u8; 20], revm::precompile::Precompile> =
+            data.precompiles.fun.clone();
+        let is_precompiled =
+            move |_: &JsValue, args: &[JsValue], ctx: &mut Context<'_>| -> JsResult<JsValue> {
+                let val = args.get_or_undefined(0).clone();
+                let buf = from_buf(val, ctx)?;
+                let addr = bytes_to_address(buf);
+                for p in precompiles.keys() {
+                    if *p == addr.0 {
+                        return Ok(JsValue::from(true))
+                    }
+                }
+                Ok(JsValue::from(false))
+            };
+        self.ctx
+            .register_global_callable("isPrecompiled", 3, unsafe {
```

Comment:
I'd like to have a function for this in builtins.rs that takes the precompiles and installs the js function